app/src/main/report/components/Artefacts/ArtefactRow.tsx

```diff
@@ -1,18 +0,0 @@
-import * as React from "react";
```

Comment:
This class is obsolete. It has effectively been replaced by the `ReportDownloadSection` component which is used for all downloads, not just artefacts